src/main/java/org/jenkinsci/plugins/parallel_test_executor/InclusionExclusionPattern.java

```diff
@@ -0,0 +1,25 @@
+package org.jenkinsci.plugins.parallel_test_executor;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * @author Vincent Latombe <vincent@latombe.net>
```

Comment:
Do we really need an author tag if we use a VCS? I would favor removing all author tags in the Jenkins project.